src/integration/pricing/simple-pricing/SimplePricingIntegration.ts

```diff
@@ -23,6 +23,7 @@ export default class SimplePricingIntegration extends PricingIntegration<SimpleP
     return this.computePrice(transaction, consumptionData);
   }
 
+  // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Please, do not blindly disable eslint rules: you just have to remove the async and the Promise<...> to properly fix the code.

Reply:
This can't be modified.
The simple pricing extends the abstract pricing which requires async methods.